source/endpoints/devices/Device.py

```diff
@@ -25,11 +24,11 @@ class Device(Resource):
     @namespace.marshal_with(device)
     def get(self, device_id):
         """ Fetch a device given its identifier """
-        return device_database.get_device(device_id)
+        return business_logic_device.get_device_from_database_by_id(device_id)
```

Comment:
In my opinion the from_database is a little to much. 
It is up to the business logic to decide if it is retrieved form a database of from somewhere else.